sdk/src/clearingHouse.ts

```diff
@@ -1297,10 +1297,10 @@ export class ClearingHouse {
 
 		if (sharesToBurn == undefined) {
 			const userAccount = this.getUserAccount();
-			const marketPosition = userAccount.perpPositions.filter((position) =>
+			const PerpPosition = userAccount.perpPositions.filter((position) =>
```

Comment:
gotta go around and change PerpPosition -> perpPosition

Reply:
ef